CMSgov/onemac
# OY2-16794: add CC emails for CHIP submissions

Story: https://qmacbis.atlassian.net/browse/OY2-16794
Endpoint: https://dr2di96l15e3f.cloudfront.net/

### Details

Pick up GH secret in prod to CC CHIP emails to CHIP Project Officers.

### Changes

- Add CC capability to email lib
- Add CC addresses from env var on CHIP-related actions

### Implementation Notes

- Env var passthrough is very clunky as is. Maybe we should try to simplify that in the future.

### Test Plan

1. Go into the AWS console and edit the environment variable `chipCcEmail` for the function `...-submit` to an email address you can access. If you are going above and beyond and testing named email addresses, make sure you **double quote** the name portion if it contains a comma.
2. Log in as `statesubmitteractive` and submit a CHIP SPA or CHIP SPA RAI. Check that email and verify that it gets CC'd on the receipt email.
3. Repeat step 1 for `...-withdraw-package`.
4. Withdraw a CHIP SPA from the package dashboard. Verify that you get another CC'd email.
